juju/juju
# audit-logging: Remove old audit observer and associated code

## Description of change

This will be replaced by the new audit logging code on the way.

## QA steps

Spin up a controller and deploy some applications, ensure that everything works as expected.
